### ISSUE TITLE
Merged entries fail to show on the update tab when new chapters are added.

### ISSUE THREAD

**Issue by Raizanad** [jobobby04/TachiyomiSY#862]
### Steps to reproduce

Make a merged entry (like Mangadex and Mangasee).
Wait.
Some entries will fail to be shown at the Update tab.

### Expected behavior

Updated chapters should show like it does with non merged entries.

### Actual behavior

They just aren't displayed from time to time and it is up to you to check if some changes happened to your reading mangas, even though they may still auto-download just fine.

### Tachiyomi version

1.9.2

### Android version

Android 10

### Device

Xiaomi Mi 8

### Other details

Haven't tested on 1.9.3, but it happens in 1.9.2 and changelog doesn't address it.

### Acknowledgements

- [X] This is a TachiyomiSY specific issue that does not happen in [Tachiyomi Preview](https://github.com/tachiyomiorg/tachiyomi/).
- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have gone through the [FAQ](https://tachiyomi.org/help/faq/) and [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[1.9.3](https://github.com/jobobby04/tachiyomisy/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by Raizanad** [comment]
As a proper example for emulation: I have a merged entry from Flame Scans and Mangadex for Omniscient. Today a chapter was added, it is downloaded and added to the library properly but it isn't on the update tab, I just know this is happening because newly downloaded entries are sorted to the top. Let me know what other details might be helpful.

**Comment by jobobby04** [comment]
Did you make this merge after the chapter was updated?

**Comment by Raizanad** [comment]
No, I have it setup my merged entries about a month ago I believe. The problem was present in 1.9.0 up to 1.9.2 and currently testing 1.9.3.
